client/src/views/asset-list.js

```diff
@@ -28,7 +28,15 @@ export default ({ assetMap, t, ...S }) => {
               {assets.map(asset =>
                 <div className="assets-table-link-row">
                   <a className="assets-table-row asset-data" href={`asset/${asset.asset_id}`}>
-                    <div className="assets-table-cell" data-label={t`Name`}>{asset.name}</div>
+                    <div className="assets-table-cell" data-label={t`Name`}>
+                      <div className="assets-table-name">
+                        {assetIcons === null ? "" : 
```

Comment:
Is it okay that the `asset-icon-placeholder` won't be added to the DOM while the icons are loading?